spec/firebase_test_runner_spec.rb

```diff
@@ -31,6 +31,13 @@
       run_tests
     end
 
+    it 'includes and properly escape the test filters if any are provided' do
```

Comment:
Nitpick: 
```suggestion
    it 'includes and properly escapes the test targets if any are provided' do
```

The "targets" suggestion is because that's how they are referred to in the code.